tests/unit/aiplatform/test_local_training_job.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
It seems like a single test file is suitable right now to test both. I suggest moving the local test to the test file above and remove this and the `test_local_training.py` file.